static/docs/commands-reference/get.md

```diff
@@ -44,4 +44,48 @@ created in the current working directory with its original file name.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-<!--  ## Example -->
+## Example
+
+Lets create a new DVC repository (requires Git):
+
+```dvc
+$ mkdir example && cd example
+$ git init
+$ dvc init
+```
+
+Now being inside a DVC project, we can use `dvc get` to download the raw dataset
+used in our
+[get started example](https://github.com/iterative/example-get-started), which
+is also a DVC project (external to the one we're in). The desired data is
+located in the `data/data.xml` path.
```

Comment:
@shcheklein 

> let's download model.pkl and we can mention the deployment scenario, or we can show actually both - model + dataset

I'm changing this to get `model.pkl` but I'm not sure what you mean by "the deployment scenario", sorry. Sounds familiar though, please remind me what you're referring to 🙂

Reply:
sure! when model is ready one of the common scenarios it to run some REST API (or sometimes ETL pipeline that involves it) to do inference (actually use ML-model to do predictions or other ML-magic). Usually it means that you need to get that binary file (model.pkl) that describes your neural-net for example to the production machine. You don't need anything else, you don't want to run `git clone`, etc. That's one of the cases where `dvc get` shines ver well. It's the same case where `dvc.api` shines very well - it's the case, but you read it into memory, from Python code. 

In certain cases you want to get some data, not model. Especially with `dvc.api`. When you need to do analysis on it.

Reply:
Got it. Mentioned in d20b190